cli/crates/gateway/src/executor.rs

```diff
@@ -106,13 +107,23 @@ impl Executor {
         let resolver_engine = UdfInvokerImpl::create_engine(self.bridge.clone());
         let search_engine = LocalSearchEngine::new(self.bridge.clone());
 
+        let mut postgres_transports = HashMap::new();
```

Comment:
This needs to be done earlier, `build_schema` is called at _every_ request because of `auth`.

Reply:
yikes. done.